Classes/Command/ResultPrinter.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace OliverKlee\Phpunit\Command;
+
+use PHPUnit_Framework_Exception;
+use PHPUnit_TextUI_ResultPrinter;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Result printer for Symfony console command.
+ *
+ * @author Felix Semmler <felix.semmler@aoe.com>
+ */
+class ResultPrinter extends PHPUnit_TextUI_ResultPrinter
+{
+    /**
+     * @var OutputInterface
+     */
+    protected $output;
+
+    /**
+     * Constructor.
+     *
+     * @param mixed      $out
+     * @param bool       $verbose
+     * @param string     $colors
+     * @param bool       $debug
+     * @param int|string $numberOfColumns
```

Comment:
How can this be both int and string?

Reply:
Ask Sebastian Bergmann ;-)
https://github.com/sebastianbergmann/phpunit/blob/master/src/TextUI/ResultPrinter.php

Reply:
ok

Reply:
ok